### PR TITLE
support --platform parameter in image_from

### PR DESCRIPTION
**Pull request by tim-vk**
fixes: #124

# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered

### CODE REVIEW

**Review by MartinBasti — APPROVED** [review]
Thank you for the contribution, this somehow failed to appear on our radar, sorry

File: dockerfile_parse/parser.py [hunk 1]
```diff
@@ -871,10 +871,12 @@ def image_from(from_value):
     :param from_value: string like "image:tag" or "image:tag AS name"
     :return: tuple of the image and stage name, e.g. ("image:tag", None)
     """
-    regex = re.compile(r"""(?xi)     # readable, case-insensitive regex
-        \s*                          # ignore leading whitespace
-        (?P<image> \S+ )             # image and optional tag
-        (?:                          # optional "AS name" clause for stage
+    regex = re.compile(r"""(?xi)        # readable, case-insensitive regex
+        \s*                             # ignore leading whitespace
+        (?P<platform> --platform=\S+)?  # optional platform parameter
```
Comment (rcerven) [inline dockerfile_parse/parser.py:876]: can you use just ? : 
(?: --platform=\S+)?

since 'platform' group isn't used anyway

Comment (tim-vk) [reply dockerfile_parse/parser.py:876]: Of ocurse it's possible: But I would think that in the future you would also want to allow to edit this platform parameter?

The current PR is just a (non-breaking) bugfix. But Ideally I think you guys should concider if this "image_from" approach is the way to go (and shouldn't be more of a "parse_FROM_keyword()" function which returns all values that are allowed within a FROM keyword.

See my other [PR](https://github.com/containerbuildsystem/dockerfile-parse/pull/129) on why I personally think that might be a good idea.

**Review by rcerven — APPROVED** [review]

**Comment by rcerven** [comment]
rn updated

**Comment by MartinBasti** [comment]
okay, for some reason, GH actions didn't run and it caused CI failures in master branch
```
./tests/test_parser.py:315:1: W293 blank line contains whitespace
4
./tests/test_parser.py:317:5: E303 too many blank lines (2)
```
https://github.com/containerbuildsystem/dockerfile-parse/actions/runs/3533487878

I'm fixing it

**Pull request merged**